commonknowledge/mapped
# Use Sentry to record job performance metrics

Logs job performance metrics to sentry using `sentry_sdk.set_measurement`

## Description
Records:
- time job waiting
- job execution time
- update rows requested
- import rows requested

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-32/updateimport-job-metrics

## How Can It Be Tested?
Import / Update a few times and check the metrics on Sentry are updated.
TODO: Where are these?

## How Will This Be Deployed?
Normal method.

## Screenshots (if appropriate):

## Types of changes


## Checklist:


- I've checked the spec (e.g. Figma file) and documented any divergences.
- My change requires a change to the documentation.
- I've updated the documentation accordingly.
- Replace unused checkboxes with bullet points.
